profiles/kentik_snmp/a10_networks/a10-thunder.yml

```diff
@@ -45,6 +45,18 @@ metrics:
         OID: 1.3.6.1.4.1.22610.2.4.1.3.3.0
         poll_time_sec: 60
         tag: CPU
+      #Average A10 Load Balancer Average Control CPU Usag
+      - name: axSysAverageCtlCpuUsage
+        OID: 1.3.6.1.4.1.22610.2.4.1.3.4.0
+        poll_time_sec: 60
+      #Average A10 Load Balancer Average Data CPU Usag
+      - name: axSysAverageDataCpuUsage
+        OID: 1.3.6.1.4.1.22610.2.4.1.3.5.0
+        poll_time_sec: 60
+      #Average A10 Load Balancer CPU Usag
+      - name: axSysCpuUsage
+        OID: 1.3.6.1.4.1.22610.2.4.1.3.2.1.3
+        poll_time_sec: 60
```

Comment:
this metric is a tabular OID that is meant to show usage per CPU, identified by their individual index values.

if this is a metric you really need, it needs to follow the MIB table syntax and root at the [axSysCpuTable](https://www.circitor.fr/Mibs/Html/A/A10-AX-MIB.php#axSysCpuTable)